src/content/app/tools/blast/components/blast-settings/BlastSettings.scss

```diff
@@ -54,8 +56,8 @@
   padding-top: 20px;
```

Comment:
Update `gap: 20px`?

Reply:
Not sure I understand what you mean here.

Reply:
gap as per XD is 20px. Its on line 55.

Reply:
I've updated the gap.

There is one thing in which the result deviates from XD: the select box for gap penalties isn't perfectly aligned with the select boxes above it on a narrow screen:

https://user-images.githubusercontent.com/6834224/234836350-366d15f6-b7ea-4d21-b3fe-60771f6e5c65.mov

There isn't a clean solution. The best I could come up with was to set a certain width (190px) on the div containing the the gap penalties; but it is fragile. I asked Andrea if we can ignore this problem, and she agreed.